sharding-distribution/shardingsphere-src-distribution/pom.xml

```diff
@@ -179,17 +179,27 @@
         </dependency>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>sharding-orchestration-core</artifactId>
+            <artifactId>sharding-orchestration-core-new</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>sharding-orchestration-reg-api</artifactId>
+            <artifactId>sharding-orchestration-center-api</artifactId>
```

Comment:
Why not use
sharding-orchestration-center
It includes :

sharding-orchestration-center-api
sharding-orchestration-center-apollo
sharding-orchestration-center-zookeeper-curator
sharding-orchestration-center-nacos